drivers/ethernet/eth_adin2111.c

```diff
@@ -852,6 +1000,11 @@ static int adin2111_init(const struct device *dev)
 		return ret;
 	}
 
+	if (ctx->oa) {
+		/* OA mode, device needs some time to respond */
+		k_msleep(1);
```

Comment:
"some time" feels too vague here (magic sleep), isn't there a minimum value specified in the datasheet?